Rakefile

```diff
@@ -19,3 +19,23 @@ desc 'Starts up browser window (requires running server)'
 task :browser do
   sh "xdg-open http://localhost:4000/index.html"
 end
+
+desc "Given a title as an argument, create a new post file"
+task :new_post, [:title] do |t, args|
+  filename = "#{Time.now.strftime('%Y-%m-%d')}-#{args.title.gsub(/\s/, '_').downcase}.md"
+  path = File.join("_posts", filename)
+  if File.exist? path; raise RuntimeError.new("#{path} exists! Refusing to overwrite"); end
+  File.open(path, 'w') do |file|
+    file.write <<-EOS
+---
+layout: post
+title: #{args.title}
+date: #{Time.now.strftime('%Y-%m-%d %k:%M:%S')}
+author: Foreman
```

Comment:
To make a guess at the current author? I'd expect that to nearly always be wrong - on my system:
```
Etc.getpwnam(Etc.getlogin).name
=> "greg"
```
I think we're just overcomplicating the task when it's something people will need to edit anyway (and they'll have to edit the metadata anyway for tags and such). I've added a "check the author" step to the README, which I think is sufficient - will that do for you?